### ISSUE TITLE
The "Status Filter" setting is ignored

### ISSUE THREAD

**Issue by nitrocode** [jan-molak/jenkins-build-monitor-plugin#257]
Hi. I have around 30 jobs that I'd like to display on the view and only show the projects that are enabled. 

Sometimes we have to disable a job and we don't want that to show up on the screen.

Thanks!

**Comment by jan-molak** [comment]
Hey @nitrocode , are you referring to the [View Job Filters](https://wiki.jenkins-ci.org/display/JENKINS/View+Job+Filters) "Status Filter" setting?

**Comment by nitrocode** [comment]
Yes! It works when I use a normal view using that plugin but it does not work in the build monitor view. I guess this is more of a feature request.
